doc/rbd/iscsi-monitoring.rst

```diff
@@ -18,41 +18,35 @@ metrics.
 
 **Installing:**
 
-1. As ``root``, enable the Ceph tools repository:
-
-   ::
-
-       # subscription-manager repos --enable=rhel-7-server-ceph-2-tools-rpms
-
-2. As ``root``, install the ``ceph-iscsi-tools`` package on each iSCSI
+1. As ``root``, install the ``ceph-iscsi-tools`` package on each iSCSI
```

Comment:
Yeah, I tried using "#.", and prefer to, instead of using the numbers, but it was renumbering them incorrectly when I was doing my test builds for some reason.